cl/recap/tasks.py

```diff
@@ -424,6 +432,7 @@ def process_recap_zip(self, pk):
         # Returning the tasks allows tests to wait() for the PDFs to complete
         # before checking assertions.
         return {
+            "new_pqs": new_pqs,
```

Comment:
Typo?

Reply:
Yes, most likely when I was copying things over. Surprised that was not found re linter, etc.